packages/flutter_tools/templates/create/ios.tmpl/Podfile

```diff
@@ -0,0 +1,10 @@
+# Uncomment this line to define a global platform for your project
+# platform :ios, '9.0'
+
+target 'Runner' do
+  # Uncomment this line if you're using Swift or would like to use dynamic frameworks
+  # use_frameworks!
```

Comment:
I am not sure I understand what exactly this does but the Flutter engine is itself a framework.